api/base_controller.go

```diff
@@ -218,9 +213,29 @@ func (c *BaseController) CreateObject(r *web.Request) (*web.Response, error) {
 		return nil, err
 	}
 
+	if createdObj.GetLastOperation().IsAsyncResponse() {
+		if err := c.checkAsyncSupport(); err != nil {
```

Comment:
same here, you check it in c.executeAsync function

Reply:
yes its relevant for all areas, it was there before but actually seems to be redundant. we will fix it everywhere.